python/hail/utils/misc.py

```diff
@@ -120,6 +120,14 @@ def new_local_temp_dir(suffix=None, prefix=None, dir=None):
     atexit.register(shutil.rmtree, local_temp_dir)
     return local_temp_dir
 
+
+def new_local_temp_path_uri(filename="temp"):
+    local_temp_dir = new_local_temp_dir()
+    path = local_temp_dir + "/" + filename
+    uri = local_path_uri(path)
+    return path, uri
```

Comment:
This is too much.  What about `new_local_temp_file`?

Reply:
Great